feature/home/src/main/java/com/epmedu/animeal/home/presentation/ui/HomeMapbox.kt

```diff
@@ -88,8 +88,10 @@ private fun MapboxMap(
         )
     }
 
-    LaunchedEffect(key1 = state.currentLocation) {
-        mapboxMapView.setLocation(state.currentLocation)
+    LaunchedEffect(key1 = state.locationState) {
+        if (state.locationState.isInitial || state.locationState.isUndefined) {
```

Comment:
Probably not a blocker, but why do we set a user location if it is undefined ?

Reply:
Undefined means that we never received a location before, so, the currentLocation value will be Tbilisi city center
Initial location is the first user location and at this point we should show user his location.
Other locations is just some sort of clarification

Reply:
Understood, thanks for the explanation!